terraform/azure/main.tf

```diff
@@ -0,0 +1,108 @@
+# AKS
+
+variable "client_id" {}
+variable "client_secret" {}
+
+resource "azurerm_resource_group" "ckan_cloud_k8" {
+  name     = "CkanCLoudk8Test"
```

Comment:
see in the AWS branch how we namespace the resource names to allow multiple build to run concurrently

Reply:
This is the resource group name, it can be shared for multiple clusters. I've updated the cluster names as in AWS.